lstchain/scripts/longterm_dl1_check.py

```diff
@@ -944,6 +948,11 @@ def plot(filename='longterm_dl1_check.h5', batch=False, tel_id=1):
                                )
     r30_lowlim = reference_rate_above_30 * (1 - rate_above_30_tolerance)
     r30_upplim = reference_rate_above_30 * (1 + rate_above_30_tolerance)
+    
+    # The outermost pixels, on the camera corners, just because of their position, have 
+    # lower values of this rate of >30 pe pulses. We just set a looser lower limit for
+    # them, to avoid lots of unnecessary warnings:
+    r30_lowlim[:, pixel_index_limit:] = 0.5 * r30_lowlim[:, pixel_index_limit:]
```

Comment:
is the limit is usually ~half of the inner ones or it is just a very relaxed one? I am afraid we may lose some real problems with those pixels because of this condition.

Reply:
The limit is half the value it would have for those pixels. It would still detect a pixel severely malfunctioning (and this is just one of many tests, surely a significant problem with a pixel would show up in other tests). All these limits will have to be refined with experience - and with the improvement of the telescope stability.